src/components/content.js

```diff
@@ -698,15 +699,27 @@ class Content extends React.Component {
     if (this.props.readOnly) return
     if (!this.isInEditor(event.target)) return
 
-    event.preventDefault()
-    const data = getTransferData(event.clipboardData)
+    const handleData = (data) => {
```

Comment:
Thanks for your suggestions. I produced a corresponding version and hope we can solve this now!